### ISSUE TITLE
DeleteColumn for a range

### ISSUE THREAD

**Issue by protist** [chrisbra/csv.vim#104]
It would be great if we could select a range of lines, then run `:DeleteColumn` or `:CSVDeleteColumn` to only act on those lines.

(I'm currently pasting data from a multi-page pdf, but obviously the copying isn't perfect, so I need to delete columns a page at a time.)

**Comment by chrisbra** [comment]
So something like `:DeleteColumn 2-4`?

**Comment by protist** [comment]
I don't think so, but I'm not sure. That command would delete all three columns in the whole file, right? What I mean is to select columns in specific *lines*, e.g. those selected with `Shift` + `V`.

e.g. select lines 5–8 with `5ggV3j`, then `:DeleteColumn 2` to delete the second column, but only in those four lines.

**Comment by chrisbra** [comment]
Hm, you could transpose your csv file and then delete those columns and then transpose it back.

**Comment by protist** [comment]
What brought this to my attention, was when I tried to copy-paste from a PDF. The imperfect copy meant that each PDF page had a variable number of columns. My thought was to selectively delete columns *only for the last paste*, so that I could normalise the number of columns in the final csv file.

I had just presumed that this would have worked in csv.vim. However, the lack of this function in csv.vim is certainly not a deal breaker, and there are definitely other ways to do it, e.g. paste into separate files, pipe through `cut` (or `awk`), etc., but I just thought it might be something useful for csv.vim to do. I'm not sure if my use case is a bit obscure, and if it's not particularly useful to others, so if you don't think it's very important, I'm happy to just close this. I just thought I'd bring it to your attention. Cheers.
